client/src/common/components/GeoLocation/utils.tsx

```diff
@@ -0,0 +1,29 @@
+/* global GeolocationPositionError */
```

Comment:
comment needed?

Reply:
Yes, otherwise I get an error. So I looked into it, and this was one of the solutions.

Reply:
what is the error?

Reply:
Eslint throws "'GeolocationPositionError' is not defined." I've searched quite a lot about it, but could not understand what I have to do. I've read like it should be available by default or something.

1. https://stackoverflow.com/questions/70759484/geolocationpositionerror-is-not-defined
2. https://stackoverflow.com/questions/69345470/how-can-i-bypass-eslint-calling-the-geolocation-interface-undefined-in-typescrip
3. https://stackoverflow.com/questions/67870306/geolocationpositionerror-code-1-message-user-denied-geolocation-prompt
There we're more stuff, but I don't remember where I've read about it.

Reply:
this error breaks your project compilation command or it is shown as warning in vscode?

Reply:
Just checked it. It compiles successfully. Just shows up as red error warning in vscode, and marks files as red. I don't like it :D

Reply:
you can find reason here, so no need to fix it in code, if you want - fix in vscode:
https://stackoverflow.com/a/66005573

Reply:
Yes, I remember seeing this comment. I have typescript@4.6.4. Can I leave that comment for the time being, so that eslint would shush?

Reply:
you can leave it with comment that it's an IDE error and promise to configure your vscode :)